sensor/common/registry/registry_store.go

```diff
@@ -84,7 +85,7 @@ func NewRegistryStore(checkTLS CheckTLS) *Store {
 	store := &Store{
 		factory:                     regFactory,
 		store:                       make(map[string]registries.Set),
-		checkTLSFunc:                tlscheck.CheckTLS,
+		checkTLSFunc:                tlscheck.CheckTLSHardcoded,
```

Comment:
hardcoded value

Reply:
[9ea428f](https://github.com/stackrox/stackrox/pull/11235/commits/9ea428ff393fc917c19d78292359f3b5f0aff0e5) this should handle that